samples/excel/89-preview-apis/style.yaml

```diff
@@ -0,0 +1,241 @@
+order: 10
+id: excel-style
+name: Style
+description: 'Add, apply, get and delete styles.'
+author: siewmoi
+host: EXCEL
+api_set:
+    ExcelApi: 1.7
+script:
+    content: |+
+        $("#setup").click(() => tryCatch(setup));
+        $("#add-new-style").click(() => tryCatch(addNewStyle));
+        $("#apply-new-style").click(() => tryCatch(applyNewStyle));
+        $("#apply-built-in-style").click(() => tryCatch(applyBuiltInStyle));
+        $("#get-style-font").click(() => tryCatch(getStyleFontProperties));
+        $("#get-style-alignment").click(() => tryCatch(getStyleAlignmentProperties));
+        $("#delete-new-style").click(() => tryCatch(deleteNewStyle));
+
+        async function addNewStyle() {
+            await Excel.run(async (context) => {
+                let styles = context.workbook.styles;
+
+                // Add a new style to the style collection.
+                // Styles is in the Home tab ribbon.
+                styles.add("Diagonal Orientation Style");
+                styles.getItem("Diagonal Orientation Style").orientation = 38;
+                styles.getItem("Diagonal Orientation Style").addIndent = true;
+               
+                let newStyle = styles.getItem("Diagonal Orientation Style");
+                context.load(newStyle);
```

Comment:
The context.load(newStyle); is not needed because you are not reading the new style's properties, only writing to them. The function works fine if I comment the line out.